docs/src/pages/[platform]/theming/style-props/examples/BackgroundExamples.tsx

```diff
@@ -0,0 +1,25 @@
+import { Card, useTheme } from '@aws-amplify/ui-react';
+
+// use any CSS value
+export const BackgroundStylePropExample = () => {
+  return (
+    <Card backgroundColor="rgb(4, 125, 149)">Background Styling Example</Card>
```

Comment:
Similar suggestion as below, setting the text color here makes it readable on both light and dark mode, since the background color stays the same in this case.

```suggestion
    <Card color="#fff" backgroundColor="rgb(4, 125, 149)">
```